File: routes/adminRoutes.py
from fastapi import APIRouter
from controllers.admin_controller import translate_admin_text, format_voice_transcription

router = APIRouter(
    prefix="/api/v1/admin",
    tags=["Admin AI"]
)

# FastAPI will automatically read the Pydantic models from the controller functions!
router.post("/translate")(translate_admin_text)
router.post("/format-voice")(format_voice_transcription)

File: routes/adminRoutes.js
import express from 'express';
import { translateAdminText, formatVoiceTranscription } from '../controllers/adminController.js';

const router = express.Router();

// Admin Dashboard AI Routes
router.post('/translate', translateAdminText);
router.post('/format-voice', formatVoiceTranscription);

export default router;